src/main/java/org/graylog/aws/inputs/transports/KinesisTransport.java

```diff
@@ -90,21 +160,34 @@ public void launch(MessageInput input) throws MisfireException {
                 awsConfig,
                 authProvider,
                 nodeId,
-                graylogConfiguration.getHttpProxyUri() == null ? null : HttpUrl.get(graylogConfiguration.getHttpProxyUri())
+                graylogConfiguration.getHttpProxyUri() == null ? null : HttpUrl.get(graylogConfiguration.getHttpProxyUri()),
+                this,
+                objectMapper, configuration.intIsSet(CK_KINESIS_MAX_THROTTLED_WAIT_MS) ? configuration.getInt(CK_KINESIS_MAX_THROTTLED_WAIT_MS) : null,
+                configuration.getInt(CK_KINESIS_RECORD_BATCH_SIZE, DEFAULT_THROTTLED_WAIT_MS)
```

Comment:
I guess this should use `DEFAULT_BATCH_SIZE` as default argument?

```suggestion
                configuration.getInt(CK_KINESIS_RECORD_BATCH_SIZE, DEFAULT_BATCH_SIZE)
```

Reply:
Agreed.